webpack.config.js

```diff
@@ -1,4 +1,6 @@
+const path = require("path");
 module.exports = {
+  mode: "development",
```

Comment:
I don't think we need mode: "development" here as it is done in the package.json script webpack:dev. Sorry if I confused you with this.

We are now relying on the webpack build going to the public folder, we still need to add the output lines to this config script.
`  output: {
    path: path.join(__dirname, 'public'),
    filename: 'main.js',
  },`